.circleci/config.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.1.0
+
+workflows:
+  package-and-push-chart-on-tag:
+    jobs:
+      - architect/push-to-app-catalog:
+          name: "package and push elastic-logging chart"
+          app_catalog: "giantswarm-incubator-catalog"
+          app_catalog_test: "giantswarm-incubator-catalog"
+          chart: "elastic-logging"
```

Comment:
This won't work. We designed it in a way that chart name must be prefixed with repository name. Details here https://github.com/giantswarm/giantswarm/issues/6025#issuecomment-498638648

Reply:
Got it and yes the validation here makes sense to me.

The problem is I find the `kubernetes-` prefix in https://github.com/giantswarm/kubernetes-elastic-stack really ugly and outdated.

@puja108 @sslavic @pipo02mix Let's chat in slack and see if there is a way to remove that prefix.

Reply:
You can rename/create another repo. I don't see this being and issue. But maybe take some time to discuss a common pattern for apps repos. Feel free to involve me in those discussions.

Reply:
Yes @pipo02mix has created a meeting for tomorrow morning. I think we can handle it within Batman but we'll keep you updated.

For now we are going to get this working for `kubernetes-elastic-stack`. So we'll need to rename this so the validation passes. cc @sslavic

Reply:
`elastic-logging` chart has been renamed to match `architect` enforced conventions (repo name was added as chart name prefix). CircleCI `architect-orb` `chart` parameter configuration has been adjusted accordingly. 

Additionally, architect and architect-orb bugs have been fixed and CircleCI configuration updated to use latest architect-orb release with fixes included. Packaging and publishing of EFK chart works now. PR branch has been published to [incubator-test catalog](https://github.com/giantswarm/giantswarm-incubator-test-catalog); tag will be released to incubator catalog upon merging PR + tagging release.

Note: there are two packages published in incubator-test catalog out of this single branch, one initially created with buggy architect-orb, and one with latest. Two can be distinguished only by different commit SHA, (and e.g. not by created date) since architect-orb indexing step reindexes all packages and not just newly added - it could be this is due to `helm repo index` limitation. On master/tag publishing to non-test catalog, version of the package will be more meaningful / human friendly, since it will be derived from tag, and there should be only one chart package per tag/version.

@rossf7 @kopiczko PTAL